Automation/00_hydra/Sweden.R

```diff
@@ -55,6 +46,7 @@ date_f <- read_xlsx(data_source, sheet = 1) %>%
 db_drive <- get_country_inputDB("SE")
 
 last_date_drive <- db_drive %>% 
+  filter(Measure %in% "Deaths") %>% 
```

Comment:
`%in% c("Cases","Deaths")` ??

Reply:
yes, that works, but since cases and deaths are always updated simulataneously, it shouldn't really matter